build_js.sh

```diff
@@ -7,6 +7,21 @@ git submodule update --init --recursive
 
 DATA_DIR="$(pwd)/data/"
 
+CORRADE_RC_PATH="$(pwd)"/build_corrade-rc/deps/corrade/src/Corrade/Utility/
+
+mkdir -p build_corrade-rc
+pushd build_corrade-rc
+cmake ../src \
+    -DBUILD_GUI_VIEWERS=OFF \
+    -DBUILD_PYTHON_BINDINGS=OFF \
+    -DBUILD_ASSIMP_SUPPORT=OFF \
+    -DBUILD_DATATOOL=OFF \
+    -DBUILD_PTEX_SUPPORT=OFF
+cmake --build . --target corrade-rc
+popd
+
+export PATH=${CORRADE_RC_PATH}:${PATH}
```

Comment:
Why not passing `-DCORRADE_RC_EXECUTABLE="$(pwd)"/build_corrade-rc/deps/corrade/src/Corrade/Utility/corrade-rc` directly to CMake instead? Modifying `PATH` might do unwanted things, especially when Emscripten is involved.